api/services/introspection/v1/introspection.proto

```diff
@@ -110,4 +114,11 @@ message DeprecationWarning {
 	string id = 1;
 	string message = 2;
 	google.protobuf.Timestamp last_occurrence = 3;
-}
\ No newline at end of file
+}
+
+message RuntimeRequest {
+	string runtime_path = 1;
+	// Options correspond to CreateTaskRequest.options.
+	// This is needed to pass the runc binary path, etc.
+	google.protobuf.Any options = 2;
+}
```

Comment:
@dmcgowan The info depends on the runc binary path, so it can't be a "state" of the plugin

Reply:
@dmcgowan WDYT?

Reply:
ping @dmcgowan 🙏

Reply:
Proposed API update https://github.com/containerd/containerd/compare/main...dmcgowan:runtime-plugin-info